martor/settings.py

```diff
@@ -4,7 +4,6 @@
 # Input: string boolean, `true/false`
 MARTOR_ENABLE_CONFIGS = getattr(
     settings, 'MARTOR_ENABLE_CONFIGS', {
-        'emoji': 'true',        # to enable/disable emoji icons.
         'imgur': 'true',        # to enable/disable imgur/custom uploader.
```

Comment:
Should we also need to remove `imgur` and `mention`?

Reply:
They're actually being used - imgur is actually generic custom image uploader and mention is for allowing search when you have `[user:<cursor>]`.

Reply:
If they are being used, is it safe to delete the extensions?

Reply:
Unless I am mistaken, I believe extensions are only used if we use their markdownify function (https://github.com/DMOJ/online-judge/blob/master/martor/utils.py#L29), which I don't forsee any plans to. Other than that, they are unused.